scene/gui/text_edit.cpp

```diff
@@ -2040,6 +2040,7 @@ void TextEdit::_gui_input(const Ref<InputEvent> &p_gui_input) {
 
 						_confirm_completion();
 						accept_event();
+						emit_signal("completion_confirmed");
```

Comment:
The fix looks fine, but wouldn't it be better to just emit "request_completion" without adding another signal?